docs/userguides/accounts.md

```diff
@@ -158,6 +180,39 @@ ape accounts import <ALIAS> --use-mnemonic --hd-path <HDPATH>
 
 If you use the `--hd-path` option, you will need to pass the [HDPath](https://help.myetherwallet.com/en/articles/5867305-hd-wallets-and-derivation-paths) you'd like to use as an argument in the command.
 If you do not use the `--hd-path` option, Ape will use the default HDPath of (Ethereum network, first account).
+
+You can script account imports using a seed phrase:
```

Comment:
```suggestion
You can [import](add link here...) an account programmatically using a seed phrase:
```

Reply:
What would the link be to?

Reply:
Assuming it's the new autodoc page, I have one in there now.  Updated using your phrasing.